assembl/lib/read_write_session.py

```diff
@@ -0,0 +1,34 @@
+# from flask.ext.sqlalchemy import SQLAlchemy, get_state
+import sqlalchemy.orm as orm
+from functools import partial
+from .logging import getLogger
+
+log = getLogger()
+
+
+class ReadWriteSession(orm.Session):
+    """A session that can divert read queries to a different engine
+    Inspired by https://gist.github.com/adhorn/b84dc47175259992d406
+    """
+
+    def __init__(self, bind=None, autoflush=False,
+                 read_bind=None, reading=False, **options):
+        self.read_bind = read_bind
+        self.reading = reading
+        orm.Session.__init__(
```

Comment:
Use super() instead of orm.Session

Reply:
`super().__init__` is for python 3. For python 2, you will need super(ReadWriteSession, self).__init__